jsonschema_gentypes/cli.py

```diff
@@ -47,6 +47,7 @@ def main() -> None:
             config = validate.load(args.config, json.loads(schema_data))
         except validate.ValidationError as error:
             LOG.error(error)
+            config = error.data
```

Comment:
and ? you don't do anything with these data ? It's just to be able to debug ?

Reply:
I set the config that's used at line 54 and next, the skip_config_errors was just not working before...

Reply:
Ah yes, sorry